doc(release): add release note for centreon-autodiscovery-21.04.4 (#1416)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-21.04/releases/centreon-commercial-extensions.md

## Centreon Auto Discovery

### 21.04.4

> Release date: `null`

Reviewer: ```suggestion
Release date: `April 1, 2022`
```

#### Bug fixes

> - Changed the size of a column to allow storing host discovery provider coming from Plugin Packs with names > 50 characters.

Reviewer 2: ```suggestion
- Changed the size of a column to allow storing host discovery providers coming from Plugin Packs with names exceeding 50 characters
```

> - ACL were not updated for items discovered by Host Discovery or Service Discovery.

Reviewer 2: ```suggestion
- ACLs were not updated for items discovered by Host Discovery or Service Discovery
```

- Fixed a bug that duplicated custom text in mappers' sources

> - Fixed an issue that caused Service Discovery scans to fail because the wrong message was caught.

Reviewer 2: ```suggestion
- Fixed an issue that caused Service Discovery scans to fail because the wrong message was caught
```

#### Security fixes

> - Fixed a SQL injection issue  in Service Discovery rules management.

Reviewer 2: ```suggestion
- Fixed an SQL injection issue in Service Discovery rules management
```
